src/lib.rs

```diff
@@ -48,7 +48,7 @@ extern crate bincode;
 extern crate config_file_handler;
 #[macro_use]
 extern crate lazy_static;
-#[macro_use]
+#[cfg_attr(test, macro_use)]
```

Comment:
Why is this change needed ? (Is it something to do with the treatment of macros in newer rusts ? )

(On a side note: Pls rebase against latest master - should be good to merge after that)